http4s/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/Config.scala

```diff
@@ -0,0 +1,293 @@
+package com.snowplowanalytics.snowplow.collectors.scalastream
+
+import java.nio.file.{Files, Path}
+
+import scala.concurrent.duration.FiniteDuration
+import scala.collection.JavaConverters._
+
+import com.typesafe.config.ConfigFactory
+
+import io.circe.config.syntax._
+import io.circe.generic.semiauto._
+import io.circe.Decoder
+import io.circe._
+
+import cats.implicits._
+
+import cats.effect.Sync
+
+import com.monovore.decline.{Command, Opts}
+
+object Config {
+
+  def parse[F[_]: Sync](
+    args: List[String]
+  ): F[Config] = {
+    val appConfig = Opts.option[Path]("config", "Path to HOCON configuration", "c", "config.hocon")
```

Comment:
For the existing collector, the `--config` param is optional.  So that you can configure the collector using system properties as an alternative to providing a file.

e.g.

```
java -Dcollector.streams.good=good -jar collector.jar
```

But here I think you've made it a required parameter.